source/api_documentation/transactions_api/advertiser_user.rst

```diff
@@ -25,7 +25,7 @@ The API follows REST conventions. Perform an HTTPS GET to the URL with the forma
     - Returns a JSON array of transaction objects. ``https://mynetwork.invoca.net/api/@@TRANSACTION_API_VERSION/advertisers/transactions/33.json``
 
 
-.. include:: intro.rst
+.. include:: _intro.rst
```

Comment:
Currently, I do not believe that there are any partials that are not .rst. Should we make it standard to make all of them .rst? My regular expression for error suppression currently assumes .rst